server/api/documents.js

```diff
@@ -29,6 +29,14 @@ router.post(
       format,
       content
     };
+    if (document.name === null || document.name === "") {
+      res.status(400).send("Some mandatory field is missing");
```

Comment:
You will need to use the keyword `return` before res.status... otherwise the execution will continue.